app/forms/idv/consent_form.rb

```diff
@@ -6,7 +6,7 @@ class ConsentForm
               acceptance: { message: proc { I18n.t('errors.doc_auth.consent_form') } }
 
     def submit(params)
-      @ial2_consent_given = params[:ial2_consent_given] == 'true'
+      @ial2_consent_given = params[:ial2_consent_given] == '1'
```

Comment:
Or is this just an artifact of moving to the built-in `check_box` helper from a plain tag

Reply:
> Or is this just an artifact of moving to the built-in `check_box` helper from a plain tag

Yeah, SimpleForm internally uses `check_box`, but it provides checked and unchecked values [defaulting to `1` and `0` respectively](https://github.com/heartcombo/simple_form/blob/86429bceb950096df3c29616f31bd5a5ce706c06/lib/simple_form/inputs/boolean_input.rb#L95-L101). We could pass those values explicitly if we wanted to avoid a change here, though it felt simpler overall to just update this to the new value.